charts/flyte-core/templates/agent/service.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.flyteagent.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "flyteagent.name" . }}
+  namespace: {{ template "flyte.namespace" . }}
+  labels: {{ include "flyteagent.labels" . | nindent 4 }}
+  {{- with .Values.flyteagent.service.annotations }}
+  annotations: {{ tpl (toYaml .) $ | nindent 4 }}
+  {{- end }}
+spec:
+  {{- with .Values.flyteagent.service.type}}
+  type: {{ . }}
+  {{- end }}
+  ports:
+  - name: agent-grpc
+    port: 8000
```

Comment:
Can we make this configurable in values chart and also use in the deployment yaml